ironfish/src/wallet/wallet.test.slow.ts

```diff
@@ -617,6 +619,96 @@ describe('Accounts', () => {
     })
   })
 
+  it('View only accounts can observe received and spent notes', async () => {
+    // Initialize the database and chain
+    const strategy = nodeTest.strategy
+    const node = nodeTest.node
+    const chain = nodeTest.chain
+
+    const account = await useAccountFixture(node.wallet, 'test')
+    const accountValue = {
+      id: 'abc123',
+      name: 'viewonly',
+      version: 1,
+      spendingKey: null,
+      publicAddress: account.publicAddress,
+      viewKey: account.viewKey,
+      outgoingViewKey: account.outgoingViewKey,
+      incomingViewKey: account.incomingViewKey,
+    }
+    const viewOnlyAccount = new Account({
+      ...accountValue,
+      walletDb: node.wallet.walletDb,
+    })
+
+    await node.wallet.walletDb.db.transaction(async (tx) => {
+      await node.wallet.walletDb.setAccount(viewOnlyAccount, tx)
+      await node.wallet.skipRescan(viewOnlyAccount, tx)
+    })
+    // TODO(joe): remove this reference once `importAccount` supports view only wallet
+    // and change accounts back to protected
```

Comment:
Currently `wallet.importAccount` doesn't support view only wallets, so I hacked this test until we merge that PR.